src/bot.ml

```diff
@@ -59,6 +59,15 @@ let callback _conn req body =
     then Some (Str.matched_group 1 body |> extract_minimize_file)
     else None
   in
+  let strip_quoted_bot_name body =
+    (* If someone says "`@coqbot minimize foo`", (with backticks), we
+       don't want to treat that as them tagging coqbot, so we adjust
+       the tagging to "@`coqbot minimize foo`" so that the matching
+       below doesn't pick up the name *)
+    Str.global_replace
+      (Str.regexp_string (f "`@%s " bot_name))
+      (f "@`%s " bot_name) body
```

Comment:
Ideally we would go further and come up with a parser for coqbot request that doesn't look inside quoted code. But well, this is good enough for now.